point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java

```diff
@@ -45,21 +59,23 @@
 @RestController
 public class FrontendController {
 
-  private static final Logger LOGGER =
-      LogManager.getLogger(FrontendController.class);
+  private static final Logger LOGGER = LogManager.getLogger(FrontendController.class);
+  private static final String FAILED = "Failed to fetch items; try again after a while";
   private static final String INVENTORY_EP_ENV = "INVENTORY_EP";
   private static final String PAYMENTS_EP_ENV = "PAYMENTS_EP";
-  private static final String FAILED = "Failed to fetch items; try again after a while";
   private static final String ITEMS_EP = "/items";
```

Comment:
where are the /items and /switch endpoints used in this service?  I'm only seeing /pay and /update

Reply:
They are used here:
- `/items`: https://github.com/GoogleCloudPlatform/anthos-edge-usecases/blob/5297f592636b96595ef73ce5685707efaaf4e4fe/point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java#L92
- `/switch`: https://github.com/GoogleCloudPlatform/anthos-edge-usecases/blob/5297f592636b96595ef73ce5685707efaaf4e4fe/point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java#L119

They are not probably visible in the PR because they were introduced in an earlier PR!